Vagrantfile

```diff
@@ -6,7 +6,7 @@ Vagrant.configure(2) do |config|
   config.vm.network :forwarded_port, guest: 9090, host: 9090
 
   config.vm.provider :virtualbox do |vb|
-    vb.customize ["modifyvm", :id, "--memory", "2048"]
+    vb.customize ["modifyvm", :id, "--memory", "4096"]
```

Comment:
This change seems unrelated to the issue?

Reply:
It is!  That's why it's its own commit :)